<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>602</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Fixedsys</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string extracomment="written by mcagriaksoy">Network Performance Test Tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="toolTip">
   <string>Network Performance Test Tool</string>
  </property>
  <property name="statusTip">
   <string>Network Performance Test Tool</string>
  </property>
  <property name="whatsThis">
   <string>Network Performance Test Tool</string>
  </property>
  <property name="accessibleName">
   <string>Network Performance Test Tool</string>
  </property>
  <property name="accessibleDescription">
   <string>Network Performance Test Tool</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>421</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>2 - I am a        </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="server_radio">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>SERVER</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="client_radio">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>CLIENT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>side.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>421</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>3 - Please,  use  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="tcp_radio">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>TCP</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="udp_radio">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>UDP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>to test.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>431</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>LOG</string>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>421</width>
         <height>421</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>1-  I want to use </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="iperf_button">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>iPerf3</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="netio_button">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>NetIO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>tool.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="stop_button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_button">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
       </font>
      </property>
      <property name="text">
       <string>Export LOG</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Advanced Setting(s)</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>431</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Advanced Settings</string>
      </property>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>90</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>no-delay</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>321</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Bandwidth (0 for ∞): </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Window Size:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Time (seconds):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>MTU Size:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="textEdit_5">
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QTextEdit" name="textEdit_6">
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="textEdit_2">
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextEdit" name="textEdit_4">
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ipv4</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>60</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ipv6</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Help</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>431</width>
        <height>551</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fixedsys'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Network Performance tool is a helper UI tool for testing your network TCP throughput and possible errors. It is created by mcagriaksoy. &lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;More information and if there is a bug, feel free to fix or open issue: &lt;a href=&quot;github.com/mcagriaksoy/networkPerfTest&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;github.com/mcagriaksoy/networkPerfTest&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; text-decoration: underline; color:#0000ff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;github.com/mcagriaksoy/networkPerfTest&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Version Info&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The IPerf tool license file:&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;quot;iperf, Copyright (c) 2014-2023, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Dept. of Energy).  All rights reserved.&amp;quot;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The NETIO tool license file: &lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;NETIO - Network Benchmark, Version 1.32 (C) 1997-2012 Kai Uwe Rommel This is a network benchmark for OS/2 2.x, Windows, Linux and Unix. It measures the net throughput of a network via TCP and UDP protocols using various different packet sizes. There is no warranty. Use this software on your own risk. Due to the complexity and variety of today's hardware and software which may be used to run these programs, I am not responsible for any damage or loss of data or hardware failures or other damage caused by correct or incorrect use of this software. It was tested very well and is expected to work correctly, but nobody can actually guarantee this under any circumstances. And because this software is essentially free (even if you register, you don't pay for using this software but for the service of getting upgrades), you get what you pay for ... Author: Kai Uwe Rommel Muenchen, Germany E-mail: rommel@ars.de &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icon.ico</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
